nss-ivory-trolls/BangazonSite
# Jb wed ticket8complete

# Ticket #8 

Given the user is authenticated
And the user is viewing their open order
When the user clicks the Complete Order button
Then the user should be presented with a view that allows them to select a payment type for the order

Given the user selects a payment option
When the user clicks the Done button
Then the payment type must be added to the order
And the user will be presented with a confirmation/thank you screen

## Type of change



# Testing Instructions for Change Made

1.  `git fetch --all`
1.  `git checkout jb-wedTicket8complete`
1.    in visual studio run Bangazon
1.   log in to bangazon with user: admin@admin.com pass: Admin8*
1.   you will now see a shopping cart in the top right of nav bar
1.   click shopping cart to see current order
1.   click "complete order"
1.   Select a payment type and click "Done"
1.   you'll now see a thank you for your order with list of orders


# Checklist:
